XamlStyler.Package/StylerPackage.cs

```diff
@@ -201,6 +209,32 @@ private void Execute(Document document)
             }
         }
 
+        private string GetConfigPathForItem(string path)
+        {
+            try
+            {
+                if (string.IsNullOrWhiteSpace(path))
+                {
+                    return null;
+                }
+
+                while ((path = Path.GetDirectoryName(path)) != null)
```

Comment:
Stop at sln level, can take a while troubleshoot a stray config in C:\.
An alternative: make it configurable via a flag in settings.